Api/Controllers/CharactersController.cs

```diff
@@ -38,6 +44,20 @@ public async Task<IActionResult> GetCharacter(Guid id)
         return Ok(character);
     }
 
+    [HttpGet("{id}/character-assets")]
+    public async Task<IActionResult> GetCharAssByCharID(Guid id)
```

Comment:
Giving the method full name will be better for readability, how the different between **Ass**ignment and **Ass**et? The method should be ListCharAssetsByCharId